homeassistant/components/switchbot_cloud/sensor.py

```diff
@@ -181,3 +197,32 @@ def _set_attributes(self) -> None:
         if not self.coordinator.data:
             return
         self._attr_native_value = self.coordinator.data.get(self.entity_description.key)
+
+
+class SwitchBotCloudSensorUsedElectricity(SwitchBotCloudSensor):
+    """Representation of a SwitchBot Cloud UsedElectricity sensor entity."""
+
+    def _set_attributes(self) -> None:
+        """Set attributes from coordinator data."""
+
+        if not self.coordinator.data:
+            return
+        used_electricity: int | None = self.coordinator.data.get(
+            self.entity_description.key
+        )
+        self._attr_native_value = used_electricity / 60000 if used_electricity else None
```

Comment:
Is the divided by 60000 for every used electricity sensor or only the EU pliug? Otherwise, I think it would be cleaner to extend the `SensorEntityDescription` by adding a `value_fn` and moving this logic to the entity descrption. This way you don't have to create a different entity class

Reply:
make sense